app/controllers/facilities_controller.rb

```diff
@@ -69,6 +69,6 @@ def set_facility
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def facility_params
-      params.require(:facility).permit(:name, :code)
+      params.require(:facility).permit(:name, :code, :organization_id)
```

Comment:
I don't think we want to add this to the params - can't we just calculate it from current_user?